bin/telemetry_sim_ATS_1VisitNight

```diff
@@ -214,8 +214,8 @@ if __name__ == "__main__":
 
         # end telemetry for EL and AZ from the ATMCS.mountEncoders
         kw = {'timestamp': time.time(),
-              'elevationCalculatedAngle': args.el*1.2,
-              'azimuthCalculatedAngle': args.az*1.2}
+              'elevationCalculatedAngle': args.el*1.001,
```

Comment:
I realize you hardcoded this because you don't have the proper telemetry to put it in, however, you should just add a note saying so. People often use something like, "TODO: Update to use end value once proper telemetry is available"

Reply:
This simple script `telemetry_sim_ATS_1VisitNight` just makes up telemetry to test the creation and receiving of messages through SAL, so proper telemetry is not expected to ever happen in this piece of code. The `1.001` factor is in place to change the values between the `START` and `END` and make sure that the computed `RASTART/RAEND` and `DECSTART/DECEND` are actually different during testing. I will add WARNING to comments to clarify what's happening here.